nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -8,31 +8,33 @@ tags:
 - Configuration
 ---
 
-The behavior of each persister can be controlled by specifying one of the respective configuration sections in the `app.config` and changing available properties, or using code API. The exact same settings are exposed via `app.config` file and code configuration API.
+In NServiceBus.Azure the behavior of the AzureStoragePersister can be controlled by specifying one of the respective configuration sections in the `app.config` and changing the available properties, or by using the code API. The exact same settings are exposed via the `app.config` and the code configuration API.
+
+In NServiceBus.Persistence.AzureStorage v1 and later the only way to configure the behavior of the persister is to use the code configuration API.
```

Comment:
As far as I see, there's no section nor example for the code configuration API. Would you consider providing an example for the code configuration API? In my opinion it'd be good to really show the new API.